lib/web_ui/lib/src/engine/platform_views/content_manager.dart

```diff
@@ -152,4 +153,17 @@ class PlatformViewManager {
       content.style.width = '100%';
     }
   }
+
+  /// Clears the state. Used in tests.
+  ///
+  /// Returns the set of know view ids, so they can be cleaned up.
+  Set<int> debugClear() {
```

Comment:
Do we need this for hot reloads?

Reply:
It's needed for tests, otherwise platform views will stay in the DOM in between tests